src/PowerShell/Microsoft.WinGet.Client.Engine/Helpers/PackageManagerWrapper.cs

```diff
@@ -21,7 +22,7 @@ internal sealed class PackageManagerWrapper
     {
         private static readonly Lazy<PackageManagerWrapper> Lazy = new (() => new PackageManagerWrapper());
 
-        private PackageManager packageManager = null;
+        private PackageManager? packageManager = null;

```

Comment:
What is the point of making this nullable and then just overriding that everywhere?  Can you not do `null!` to trick it into letting you have a null value?  Maybe you need to do `((PackageManager?)null)!`.

Reply:
`null!` is the way to go